apps/fluent-tester/src/TestComponents/Button/ButtonIconTestSection.tsx

```diff
@@ -2,28 +2,30 @@ import { ButtonV1 as Button } from '@fluentui/react-native';
 import * as React from 'react';
 import { Platform, View, StyleSheet } from 'react-native';
 import { commonTestStyles, testContentRootViewStyle } from '../Common/styles';
-import { SvgIconProps } from '@fluentui-react-native/icon';
-import TestSvg from './test.svg';
+import { testImage, svgProps } from '../Common/iconExamples';
 import { SvgXml } from 'react-native-svg';
+import { IconSourcesType } from '@fluentui-react-native/icon';
+import { useTheme } from '@fluentui-react-native/theme-types';
 
 const styles = StyleSheet.create({
   chevron: { paddingStart: 4 },
 });
 
 export const ButtonIconTest: React.FunctionComponent = () => {
+  const theme = useTheme();
+
   const fontBuiltInProps = {
     fontFamily: 'Arial',
     codepoint: 0x2663,
     fontSize: 24,
   };
-
-  /* eslint-disable @typescript-eslint/no-var-requires */
-  const testImage = require('../../../../assets/icon_24x24.png');
-
-  const svgProps: SvgIconProps = {
-    src: TestSvg,
-    viewBox: '0 0 500 500',
+  const iconProps: IconSourcesType = {
+    svgSource: svgProps,
   };
+  if (Platform.OS === 'android') {
+    iconProps.color = theme.host.appearance === 'light' ? 'white' : 'black';
+  }
+
```

Comment:
Why specifically Android?

Reply:
I mentioned this above https://github.com/microsoft/fluentui-react-native/pull/2350#discussion_r1026994172